.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
Kindly delete this file
- file name ```.vscode/settings.json```
- And create a ```.gitignore``` file and then add ```.vscode/settings.json``` inside